lib/steep/drivers/check.rb

```diff
@@ -214,8 +214,8 @@ def print_result(project:, notifications:)
           total = errors.sum {|notification| notification[:diagnostics].size }
 
           errors.each do |notification|
-            path = project.relative_path(Pathname(URI.parse(notification[:uri]).path))
-            buffer = RBS::Buffer.new(name: path, content: path.read)
+            path = Pathname(URI.parse(notification[:uri]).path)
+            buffer = RBS::Buffer.new(name: project.relative_path(path), content: project.absolute_path(path).read)
```

Comment:
Nice catch!
Could you remove `project.absolute_path` call, because the path is assumed to be absolute.

Reply:
Thanks review! fix it 👍